Assets/MixedRealityToolkit/Providers/Experimental/BaseSpatialSceneObserver.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.SpatialAwareness
+{
+    public abstract class BaseSpatialSceneObserver : BaseSpatialObserver, IMixedRealitySpatialAwarenessSceneUnderstandingObserver
```

Comment:
Can you add comment here elaborating difference between BaseSpatialObserver and BaseSpatialSceneObserver ?